cmd/integration/commands/root.go

```diff
@@ -22,12 +24,28 @@ import (
 	"github.com/ledgerwatch/erigon/turbo/logging"
 )
 
+func expandHomeDir(dirpath string) string {
```

Comment:
No, manual solutions is always bad and non-crossplatform. Use stdlib, something like filepath.Abs

Reply:
There is no appropriate function in `path` or `os` packages, I've checked them all before writing my own (yes,  I can't believe it's true too:) ). 

`filepath.Abs` just adds current working dir as prefix to the path starting with `~` ([playground with example](https://go.dev/play/p/jyMZ1WfL5Yg) ). And it is the immediate error reason really,  it is called here for all pathes: https://github.com/ledgerwatch/erigon-lib/blob/7764cf6d5583547249c0365c1494f7cef9e13d32/common/datadir/dirs.go#L42,  without this call database openning with paths started with `~` works correctly. May be it is better to add this fix to `erigon-lib` really.) 

I've changed code to use `os.PathSeparator` instead of hardcoded  `/`. `strings.HasPrefix` is crossplatform enough because `filepath.HasPrefix` was deprecated because of the issues with case sensitivity,  it is not important when we are searching for `~`, so we can use it.   Also I'm using path without modifications if i can't get home directory on this platform.